pkg/trace/api/profiles.go

```diff
@@ -68,7 +74,20 @@ func (r *HTTPReceiver) profileProxyHandler() http.Handler {
 	if err != nil {
 		return errorHandler(err)
 	}
-	tags := fmt.Sprintf("host:%s,default_env:%s", r.conf.Hostname, r.conf.DefaultEnv)
+	ver, err := version.Agent()
+	verStr := "unknown"
+	if err == nil {
+		verStr = ver.GetNumberAndPre()
+	}
+
+	tags := fmt.Sprintf("host:%s,default_env:%s,agent_version:%s", r.conf.Hostname, r.conf.DefaultEnv, verStr)
+
+	// Note: check for Fargate explicitly, need identifying tags for host tracking
```

Comment:
This comment is not needed. The code is very self-explanatory and obvious.

Reply:
The "why" is less obvious, but up to you